### DISCUSSION TITLE
Punctuations

### DISCUSSION THREAD

**Discussion by Bhushannn** [deepgram/community#208] [General help]
I am not getting proper punctuations in real time transcription. sometimes full stops are added unnecessarily and are sometimes omitted, no use of commos and no use of capital letters after a full stop.

Can anyone help me please I am using Nova model for realtime transcription

**Comment by jjmaldonis** [comment]
Hey @Bhushannn, can you give us some more info about your request, such as what language you are transcribing? Also, is the punctuation only poor at the beginning of the audio (e.g. for the first 30 seconds)? Sometimes it can take a few sentences for the punctuation to take effect.

**Reply by Bhushannn** [reply]
Transcribing English. It goes for entire speech

**Reply by Bhushannn** [reply]
hey @jjmaldonis can you please tell me how can i improve on this

**Reply by jjmaldonis** [reply]
Hey @Bhushannn, if the punctuation is poor for the entire speech then I'm not sure what is going on. You can try using the `smart_format=true` option, which may improve results.

Alternatively, if you can provide the following information so that we can run the request ourselves and see the results, we may be able to help more:

1. The `request_id` of a request that is giving poor punctuation (this can be found under the Usage->Logs section of https://console.deepgram.com/)
2. The full URL, including the query parameters
3. The audio file you are transcribing
4. The full code that we can run to run the request ourselves

**Comment by Bhushannn** [comment]
1b6aa57f-95ff-4f02-bfc0-ddeabf002539
The above is the request ID.
In this, you can see that sometimes after Full Stops there is no starting with Capital Letters

Using the same codes as shown by the senior developer in the Youtube video
